index.js

```diff
@@ -1,38 +1,109 @@
-var EventEmitter = require('events').EventEmitter;
-var utils = require('util');
+'use strict';
 
-function PGEvents (client) {
-  if (!(this instanceof PGEvents)) return new PGEvents(client);
-  var self = this;
-  this.client = client;
+var EventEmitter = require('events').EventEmitter
+    , utils = require('util')
+    , fs = require('fs')
+    , path = require('path')
+    , doT = require('dot')
+    , pg = require('pg')
+    , _ = require('mol-proto')
+    , installSQLtemplate = getInstallSQLtemplate();
+
+
+var DEFAULTS = {
+    changesChannel: 'data_changes',
+    names: {
+        notifyTrigger: 'data_change_notify_trigger',
+        notifyTriggerFunc: 'data_change_notify_trigger_func',
+        addNotifyTrigger: 'add_notify_trigger_to_table',
+        removeNotifyTrigger: 'remove_notify_trigger_to_table'
+    },
+    operationEvents: true
+};
+
+function PGObserver (opts) {
+    if (!(this instanceof PGObserver)) return new PGObserver(opts);
+    this.client = opts.client || new pg.Client(opts.conString);
```

Comment:
ah, this is breaking the subscribe test. The first argument in the test script was originally just pg client.
It should now be initialized as  `self.pgevents = PGEvents({client: client})`
https://github.com/paulserraino/pg-pubsub/blob/master/tests/tests.js#L10